atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/timestamp/CleanTransactionRange.java

```diff
@@ -85,14 +88,19 @@ protected int executeTimestampCommand(AtlasDbServices services) {
                     SafeArg.of("startTs", startTs), SafeArg.of("commitTs", commitTs));
 
             Cell key = Cell.create(rowName, TransactionConstants.COMMIT_TS_COLUMN);
-            toDelete.put(key, value.getTimestamp());  //value.getTimestamp() should always be 0L
+            byte[] valueForRollbackTimestamp =
+                    TransactionConstants.getValueForTimestamp(TransactionConstants.FAILED_COMMIT_TS);
+
+            txTableValuesToWrite.put(key, valueForRollbackTimestamp);
         }
 
-        if (!toDelete.isEmpty()) {
-            kvs.delete(TransactionConstants.TRANSACTION_TABLE, toDelete);
-            printer.info("Delete completed.");
+        if (!txTableValuesToWrite.isEmpty()) {
+            Map<TableReference, Map<Cell, byte[]>> valuesToPut = new HashMap<>();
+            valuesToPut.put(TransactionConstants.TRANSACTION_TABLE, txTableValuesToWrite);
+            kvs.multiPut(valuesToPut, AtlasDbConstants.TRANSACTION_TS);
```

Comment:
We don't want to do this either. these values won't be visible in cassandra since 0 < all wall clock times (cas uses wall clock time, even though atlas sends 0)